File: QTD2.Infrastructure/Model/SimulatorScenario/SimulatorScenario_TasksResponseVM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QTD2.Infrastructure.Model.SimulatorScenario
{
    public class SimulatorScenario_TasksResponseVM
    {
        public List<SimulatorScenario_Task_VM> SimulatorScenarioTaskVMs { get; set; } = new List<SimulatorScenario_Task_VM>();
        public List<SimulatorScenario_Procedure_VM> SimulatorScenarioProcedureVMs { get; set; } = new List<SimulatorScenario_Procedure_VM>();
        public List<SimulatorScenario_EnablingObjective_VM> SimulatorScenarioEnablingObjectiveVMs { get; set; } = new List<SimulatorScenario_EnablingObjective_VM>();
    }
}

File: QTD.SPA/src/app/_DtoModels/SimulatorScenarios_New/SimulatorScenario_TasksResponseVM.ts
import { SimulatorScenario_EnablingObjective_VM } from "./SimulatorScenario_EnablingObjective_VM";
import { SimulatorScenario_Procedure_VM } from "./SimulatorScenario_Procedure_VM";
import { SimulatorScenario_Task_VM } from "./SimulatorScenario_Task_VM";

export class SimulatorScenario_TasksResponseVM{
    simulatorScenarioTaskVMs:SimulatorScenario_Task_VM[];
    simulatorScenarioProcedureVMs:SimulatorScenario_Procedure_VM[];
    simulatorScenarioEnablingObjectiveVMs:SimulatorScenario_EnablingObjective_VM[];
}